benchmarks/huffman_coding/procedural_f#/Program.fs

```diff
@@ -0,0 +1,137 @@
+﻿// Learn more about F# at http://fsharp.org
+
+open System
+open System.Collections.Generic
+open System.Text
+
+//------ HEAP
+
+let positions: Dictionary<int,int> = new Dictionary<int, int>()
+type Heap = {
+    //Type help:   (id, (freqency, Dictionary<char, encodeing>) array)
```

Comment:
```suggestion
```

Reply:
For min skyld behøves det ikke blive slettet, synes det hjælper på forståelsen